frontend/src/components/DesktopApps/Taskbar.tsx

```diff
@@ -18,6 +19,7 @@ const Taskbar: React.FC = () => {
       }}
     >
       <p>

```

Comment:
Rather than editing the text alignment in the `AutoCollector` component, try messing with `<p>`'s style here. Styles here should affect all the child text.